go.mod

```
@@ -1,6 +1,6 @@
 module github.com/lima-vm/lima
 
-go 1.21
+go 1.22.0
```

Could we use `go 1.22`?

---

No, the syntax was changed to require the patch version too.

---

Should we use the latest available patch version?
```suggestion
go 1.22.6
```

---

I don't think it is necessary.
It is user's responsibility to update Go.

---

Can we merge?

---

`go 1.22` seems to work with the current toolchains

---

CI is passing with `go 1.22`, so let me merge this